recipe/meta.yaml

```diff
@@ -28,17 +28,19 @@ requirements:
     - wheel
```

Comment:
cython could be bounded, see [here](https://github.com/rasterio/rasterio/blob/560d4dd6b6c7fb402779ac815f83d618f357d013/pyproject.toml#L2)